src/main/java/de/tum/in/www1/artemis/service/util/XmlFileUtils.java

```diff
@@ -25,16 +25,6 @@ public static Document readFromString(String xmlString) {
         return parseDocument(xmlString);
     }
 
-    /**
-     * Reads an XMl file from the resources for a given relative path
-     *
-     * @param resource The XML file resource with the data properly formatted
-     * @return The parsed XML document
-     */
-    public static Document readXmlFile(Resource resource) {
```

Comment:
why are you deleting this file? 
Don't you think it might be used in a future purpose?

Reply:
This change was done by @krusche so I assume we won't need this in the future too.